run.py

```diff
@@ -48,6 +48,10 @@ def signal_handler(sig, frame):
     halt_all()
 
 
+def get_url(endpoint):
```

Comment:
Please remove the `from grader.utils import get_url` line above since utils has been deleted.